backend/src/test/java/eventlistener/controller/EventControllerTest.java

```diff
@@ -193,4 +194,55 @@ void testGetAllEventsExcludeUser(){
         assertThat(responseEntity.getStatusCode(), is(HttpStatus.OK));
         assertThat(responseEntity.getBody(), arrayContaining(exclude));
     }
+
+    @Test
+    @DisplayName("Should Edit a given Event and return the new Event")
+    void testEditEvent(){
+        //GIVEN
+        NotificationUser user1 = NotificationUser.builder()
+                .name("Test User 1")
+                .email("test@test.de")
```

Comment:
will you allow different users with the same emails?
could lead to errors in your test in case you will build some email validation later on